.cci.jenkinsfile

```diff
@@ -20,13 +20,9 @@ pod(image: 'registry.fedoraproject.org/fedora:32', runAsUser: 0, kvm: true, memo
         """)
     }
 
-    stage("Build FCOS") {
-        shwrap("chown builder: /srv")
-        // just split into separate invocations to make it easier to see where it fails
-        cosa_cmd("init https://github.com/coreos/fedora-coreos-config")
-        cosa_cmd("fetch --strict")
-        cosa_cmd("build --strict")
-    }
+    shwrap("chown builder: /srv")
+    // Run stage Build FCOS (init, fetch and build)
+    fcosBuild(skipKola: 1, cosaDir: "/srv", noForce: 1)
```

Comment:
Also, maybe instead of 1, let's just `true`?

Reply:
In this case I'm skipping the skipKola called inside the fcosBuild that doesn't take any params,  so can call it after in the way I want.